mongoDB_ODS/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+project_name	:=	"MongoDB_ODS"
+docker_file	:=	"docker-compose.yaml"
+source_file	:=	"ml-latest-small.zip"
+logs_dir	:=	"logs"
+sandbox_ip	:=	127.0.0.1
+sandbox_port	:=	3999
+navigator_port	:=	4000
+
+all:
```

Comment:
That's more like `help`, not `all`... and might become stale, but is minor.

Reply:
No it does not, but it's illustrative and easily consumable